tests/rptest/tests/cloud_storage_scrubber_test.py

```diff
@@ -215,6 +221,41 @@ def __init__(self, test_context):
         self.rpk = RpkTool(self.redpanda)
         self.expected_error_logs = []
 
+    def _produce_inventory_report(self):
+        data = StringIO()
+        writer = csv.writer(data)
+
+        # Change segment names before adding to data set. As the segment to be dropped is computed later,
+        # at this time its name is not known. If it is present in the data set, it will not be checked in
+        # the bucket and the anomaly will never be generated. This report simply exists as a sanity check
+        # that the scrubber works as expected when the inventory data set is empty. We do not altogether
+        # skip adding the segment, because we still want the data set to be present on disk, so that the
+        # scrubber loads the data and performs segment checks. The presence of the changed segment name
+        # causes the NTP hash dir to be created on disk.
+        # TODO add more assertions once metrics for scrubber run are added
```

Comment:
+1

Can we have a version of this test that doesn't change the name?

It'd also be nice to have a test that removes segments, generates the report, and asserts that there are anomalies based on the inventory report

Reply:
Yeah, with metrics of the scrub process added it will be easier to make some assertions, right now what happens in the scrubber is a bit of a black box for ducktape, especially asserting why actions were taken by scrubber.